### PR TITLE
Fix dropdown

### PR DESCRIPTION
**Pull request by JMurphyWeb**
Sorry it was impossible to split this into one bug per PR as lots/all bugs were being caused by the same problem.

### CODE REVIEW

File: server/handlers.js [hunk 1]
```diff
@@ -53,27 +53,28 @@ handlers.viewAllUsers = (request, reply) => {
...
-              path: '/people/add',
-              name: '+'
-            }, {
-              path: '/orgs',
-              name: 'Orgs'
-            }]
-          };
-          reply.view('people/view', allUsers);
-        }
+      var orgs = stringifiedOrgs.map(element => JSON.parse(element));
```
Comment (nelsonic) [inline server/handlers.js]: @JMurphyWeb please split this handler out into a separate file and require it in the route in-line.

File: server/handlers.js [hunk 2]
```diff
@@ -210,6 +197,33 @@ handlers.createNewPrimaryUser = (request, reply) => {
   });
 };
 
+handlers.toggleArchiveUser = (request, reply) => {
```
Comment (nelsonic) [inline server/handlers.js]: This is a _**New**_ handler. it should not be in `handlers.js` anymore... we _know_ that we don't want a `handlers.js` "_Mega File_" that is a merge-conflict magnet. please create a separate file inside a `/handlers` folder and require this handler in-line in the `route` thanks. 👍

File: server/routes.js [hunk 3]
```diff
@@ -50,6 +50,11 @@ var routes = [
   },
   {
     method: 'GET',
+    path: '/people/{id}/toggle-archive',
+    handler: handlers.toggleArchiveUser
```
Comment (nelsonic) [inline server/routes.js]: should be: `handler: require('./handlers/toggleArchiveUser.js')` (_i.e. split out the file so we can move towards our goal of modularity_)

**Comment by JMurphyWeb** [comment]
@nelsonic i will leave this pr here, but it is probably easier to just compare review #148

**Pull request merged**